tests/js/spec/views/organizationDiscover/index.spec.jsx

```diff
@@ -99,7 +104,11 @@ describe('OrganizationDiscoverContainer', function() {
     it('display coming soon message', async function() {
       const organization = TestStubs.Organization({projects: [TestStubs.Project()]});
       const wrapper = mount(
-        <OrganizationDiscoverContainer location={{query: {}, search: ''}} params={{}} />,
+        <OrganizationDiscoverContainer
+          location={{query: {}, search: ''}}
+          params={{}}
+          selection={{}}
```

Comment:
Do we need tests that have projects in selection?

What happens if `params.projects` !== `selection.projects`?

Reply:
Added a test with a selection.

`params` represents the last valid query that has been run (i.e. the current results being displayed). In the case of a saved query it's also an empty object. This can be different from the current global selection (they are only the same the same upon successfully running a new query)